### PR TITLE
thin-provisioning-tools: Add thin-provisioning-tools package.

### PR DESCRIPTION
**Pull request by AkitoTakada**
0001-do-not-strip-pdata_tools-at-do_install.patch
  It is not necessary because the Debian source is already patched.
0001-fix-compile-failed-with-libc-musl.patch
use-sh-on-path.patch
  These are necessary.

### CODE REVIEW

**Comment by AkitoTakada** [comment]
I have confirmed that the file exists, but I have not tested it.
The reasons for not being able to test are as follows.
There is no logical volume in the emulator.
Also a logical volume cannot be created because there is no physical volume.
Therefore, it cannot be tested.

All of libaio, thin-provisioning-tools, and lvm2 are in the same situation.

```
root@qemuarm64:~# find /usr/sbin -name "*cache_*" | sort -n
/usr/sbin/cache_check
/usr/sbin/cache_dump
/usr/sbin/cache_metadata_size
/usr/sbin/cache_repair
/usr/sbin/cache_restore
/usr/sbin/cache_writeback
root@qemuarm64:~# find /usr/sbin -name "*era_*" | sort -n
/usr/sbin/era_check
/usr/sbin/era_dump
/usr/sbin/era_invalidate
/usr/sbin/era_restore
root@qemuarm64:~# find /usr/sbin -name "pdata_tools" | sort -n
/usr/sbin/pdata_tools
root@qemuarm64:~# find /usr/sbin -name "thin_*" | sort -n
/usr/sbin/thin_check
/usr/sbin/thin_delta
/usr/sbin/thin_dump
/usr/sbin/thin_ls
/usr/sbin/thin_metadata_size
/usr/sbin/thin_repair
/usr/sbin/thin_restore
/usr/sbin/thin_rmap
/usr/sbin/thin_trim
```

File: recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/${BPN}-${PV}"
+S = "${WORKDIR}/${BPN}-${PV}"
+
+SRC_URI += " \
+           file://0001-fix-compile-failed-with-libc-musl.patch \
+           file://use-sh-on-path.patch \
+"
+
+UPSTREAM_CHECK_GITTAGREGEX = "v(?P<pver>\d+(\.\d+)+)"
```
Comment (masami256) [inline recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb]: @AkitoTakada  This recipe doesn't checkout source code from git, can be removed.

**Comment by AkitoTakada** [comment]
Deleted as pointed out.

**Comment by yuwata** [comment]
@masami256 Also in this case, please use 'squash' merge.

**Comment by masami256** [comment]
@AkitoTakada Would you merge 3bd80c43b29f4d5ddb27a7def90c69b0913c4eaf and b43983edf071dd26ff8ba8d08699942201856e03 ? It creates simple commit for add a package.

**Comment by masami256** [comment]
@AkitoTakada commit b43983edf071dd26ff8ba8d08699942201856e03 removes use-sh-on-path.patch from SRC_URI, is it okay to remove the patch ?

File: recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# base recipe: https://github.com/openembedded/meta-openembedded/tree/warrior/meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb
```
Comment (hiraku-wfs) [inline recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb]: Could you change this line to the following?
`# base recipe: meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb`

**Comment by AkitoTakada** [comment]
I misunderstood the indication.
The patch has been returned.
The description of the base recipe has been corrected as indicated.

**Comment by masami256** [comment]
thanks! merged.

**Pull request merged**